test/createAzureResource.test.ts

```diff
@@ -154,3 +219,16 @@ function getWebsiteManagementClient(testAccount: TestAzureAccount): WebSiteManag
 function getResourceManagementClient(testAccount: TestAzureAccount): ResourceManagementClient {
     return new ResourceManagementClient(testAccount.getSubscriptionCredentials(), testAccount.getSubscriptionId());
 }
+
+async function clearnewprojectfile(projectpath: string): Promise<void> {
```

Comment:
Please camelCase function and variable names.
`clearNewProjectFile` and `projectPath`